fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationsDialogFragment.kt

```diff
@@ -69,8 +70,13 @@ class TranslationsDialogFragment : BottomSheetDialogFragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?,
     ): View = ComposeView(requireContext()).apply {
-        // Signalling user intention to translate
-        browserStore.dispatch(TranslationsAction.TranslateExpectedAction(args.sessionId))
+        // Signalling need to fetch languages
+        browserStore.dispatch(
```

Comment:
Not a permanent solution, just refactoring the dispatch away from `TranslateExpectedAction`.

Figuring out where this should be really called from in [1875817](https://bugzilla.mozilla.org/show_bug.cgi?id=1875817).